roles/ffh.mesh_tunneldigger/handlers/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Restart tunneldigger
+  systemd:
+    name: tunneldigger
+    state: restarted
+  when: domains is defined and sn is defined
+
+- name: Load kernel modules
```

Comment:
This does not actually chnage anything, does it?
kmod ran once on systemstart and since systemctl uses a sevice to manage it is never affected again unless stopped?